crypto/des/rpc_enc.c

```diff
@@ -1,30 +0,0 @@
-/*
```

Comment:
> OPENSSL_indirect_call(), which appears to be undocumented

See commentary in crypto/x86cpuid.pl. Original idea was to use it with 3rd party DLLs, those that would be loaded at run-time with LoadLibrary, without having to think about calling convention.

> and unused in the tree

That's correct. Idea got no traction. Yeah, it can be removed...